src/Main.ts

```diff
@@ -1,34 +1,41 @@
 import {ControllerSer} from './ControllerSer';
-const readline = require('readline');
+const readline = require('readline-sync');
```

Comment:
This wasn't added in package.json 
`npm i --save x` to add x as a dependencie
`npm i --save-dev x` to add x as a dev-dependencie

Reply:
Done